### PR TITLE
Get raster tiles from S3, which speeds things up a lot

### LINKED ISSUE #307: Use precomputed tile layers

### PR DESCRIPTION
**Pull request by riecke**
Notes:
1. Just a few of the rasters are still being generated.
2. The max zoom level is now 13.  There isn't much detail beyond this at 300m resolution, and it takes a lot more time to generate level 14 (probably 4 times longer)

Tested locally.

	modified:   src/interface/src/app/map/map-manager.ts
	modified:   src/interface/src/app/types/legend.types.ts
	modified:   src/planscape/config/conditions.json

closes #307

### CODE REVIEW

File: src/interface/src/app/map/map-manager.ts [hunk 1]
```diff
@@ -623,23 +623,18 @@ export class MapManager {
...
 
     let colormap = map.config.dataLayerConfig.colormap;
     if (colormap?.length === 0 || !colormap) {
       colormap = DEFAULT_COLORMAP;
     }
 
-    map.dataLayerRef = L.tileLayer.wms(
-      BackendConstants.END_POINT + '/conditions/wms',
+    map.dataLayerRef = L.tileLayer(
+      'https://planscape-tiles.s3.us-west-1.amazonaws.com/' + filepath + '/{z}/{x}/{y}.png',
```
Comment (thequeenofspades) [inline src/interface/src/app/map/map-manager.ts:633]: Can this be part of BackendConstants?

File: src/interface/src/app/types/legend.types.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const DEFAULT_COLORMAP = 'viridis';
+export const DEFAULT_COLORMAP = 'turbo';
```
Comment (thequeenofspades) [inline src/interface/src/app/types/legend.types.ts:1]: just checking my understanding: colormap is not actually used as a param for getting the rasters anymore, right?

**Review by thequeenofspades — APPROVED** [review]

File: src/interface/src/app/types/legend.types.ts (hunk 2 above)
Comment (riecke) [reply src/interface/src/app/types/legend.types.ts:1]: Yes, that's right!

File: src/interface/src/app/map/map-manager.ts (hunk 1 above)
Comment (riecke) [reply src/interface/src/app/map/map-manager.ts:633]: Good call.  I will fix that.

**Pull request merged**